src/internal/connector/data_collections.go

```diff
@@ -80,6 +80,8 @@ func (gc *GraphConnector) ProduceBackupCollections(
 
 	if !canMakeDeltaQueries {
 		logger.Ctx(ctx).Info("delta requests not available")
+
+		ctrlOpts.ToggleFeatures.DisableDelta = true
```

Comment:
This is something that I debated myself. The reason why I removed the `canMakeDeltaQueries` is because it was getting passed down quite a bit along with this which mostly carries the same(depends on how you look at it) data.

> Turning on the "I, the user, want to disable deltas" flag based on the state of data mixes concerns in ways that might be difficult to puzzle out if we're investigating state down the line.

The idea was that the log line just above this is what we would use to distinguish what caused us to not use the delta queries. I'm merging it for now, but let me know if the reasoning is not valid and I can switch it.